package.json

```
@@ -56,7 +56,7 @@
     "coveralls": "cat unit-coverage.lcov | coveralls",
     "devtools": "bash lighthouse-core/scripts/roll-to-devtools.sh",
     "chrome": "node lighthouse-core/scripts/manual-chrome-launcher.js",
-    "fast": "yarn start --emulated-form-factor=none --throttlingMethod=provided",
+    "fast": "yarn start --screenEmulation.disabled --no-emulated-user-agent --throttlingMethod=provided",
```

```suggestion
    "fast": "yarn start --preset=desktop --throttlingMethod=provided",
```

maybe captures the intention a bit better?